### PR TITLE
Fix invoke CLI to disable agent bundling

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Fix CLI used to disable agent bundling.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: omnibus/config/software/datadog-agent.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@
     command "inv -e rtloader.clean"
     command "inv -e rtloader.make --python-runtimes #{py_runtimes_arg} --install-prefix \"#{install_dir}/embedded\" --cmake-options '-DCMAKE_CXX_FLAGS:=\"-D_GLIBCXX_USE_CXX11_ABI=0 -I#{install_dir}/embedded/include\" -DCMAKE_C_FLAGS:=\"-I#{install_dir}/embedded/include\" -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_FIND_FRAMEWORK:STRING=NEVER'", :env => env
     command "inv -e rtloader.install"
-    bundle_arg = bundled_agents.map { |k| "--bundle #{k}" }.join(" ")
+    bundle_arg = bundled_agents ? bundled_agents.map { |k| "--bundle #{k}" }.join(" ") : "--bundle agent"
```
Comment (chouquette) [inline omnibus/config/software/datadog-agent.rb:83]: I'm not sure I understand the expected logic, if nothing is added to `bundled_agent` then the default behavior is  to bundle only the agent.

So far so good, but this seem to imply that the agent needs to be explicitly enabled to be built, and currently on linux `bundled_agents` will contain `process-agent` `security-agent` and `system-probe`, which should cause `--bundle agent` not to be passed to the omnibus task and the agent not to be built.

Could you clarify what the expected behavior is?

Comment (lebauce) [reply omnibus/config/software/datadog-agent.rb:83]: To disable the bundling completely, one has to use `inv agent.build --bundle agent`.
If you want to bundle other agents, you need to either:
- use the defaults specified in the "invoke" files
- explicitely define the list (this is what we do here).

**Review by hestonhoffman — APPROVED** [review]
No docs review required

**Review by chouquette — APPROVED** [review]

**Pull request merged**